cfn-resources/project-ip-access-list/cmd/resource/create.go

```diff
@@ -72,6 +73,12 @@ func createEntries(model *Model, client *util.MongoDBClient) (handler.ProgressEv
 	projectID := *model.ProjectId
 
 	if isEntryAlreadyInAccessList, err := isEntryAlreadyInAccessList(client, model); isEntryAlreadyInAccessList || err != nil {
+		if err != nil {
+			return handler.ProgressEvent{
+				Message:          fmt.Sprintf("Error validating entries: %s", err.Error()),
+				OperationStatus:  handler.Failed,
+				HandlerErrorCode: cloudformation.HandlerErrorCodeAlreadyExists}, err
```

Comment:
is it correct to return `cloudformation.HandlerErrorCodeAlreadyExists` here?

Reply:
it wont be shown to the user, but i will change it anyways